### ISSUE TITLE
Dotnet cyclonedx bom import results in blank descriptions

### ISSUE THREAD

**Issue by MatthewFergus** [DependencyTrack/dependency-track#746]
### Current Behavior:
Importing a .net bom xml generated using cyclonedx-dotnet with populated description fields results in a blank component description in dependency track.

### Steps to Reproduce:
Import the following bom.xml file to a dependency track project via the GUI:
```xml
<?xml version="1.0" encoding="utf-8"?>
<bom version="1" serialNumber="urn:uuid:09500aa9-dc62-41bb-8d53-ac0df43e87b0" xmlns="http://cyclonedx.org/schema/bom/1.1">
  <components>
    <component type="library">
      <name>Amazon.Lambda.Core</name>
      <version>1.0.0</version>
      <description><![CDATA[Amazon Lambda .NET Core support - Core package.]]></description>
      <scope>required</scope>
      <licenses>
        <license>
          <url>http://aws.amazon.com/apache2.0/</url>
        </license>
      </licenses>
      <purl>pkg:nuget/Amazon.Lambda.Core@1.0.0</purl>
      <externalReferences>
        <reference type="website">
          <url>https://aws.amazon.com/lambda/</url>
        </reference>
      </externalReferences>
    </component>
  </components>
</bom>
```

Results in a blank description:
![image](https://user-images.githubusercontent.com/27768983/88622631-8e7f8a00-d0e6-11ea-9d55-201dd3b123db.png)

### Expected Behavior:
Description should be "Amazon Lambda .NET Core support - Core package".

### Environment:
  
  - Dependency-Track Version: 3.8.0
  - Distribution: Docker
  - Browser: Chrome 

### Additional Details:

If I remove the CDATA tag the description uploads as expected

**Comment by stevespringett** [comment]
Reproduced. Thanks for reporting.

**Comment by stevespringett** [comment]
This issue is resolved simply by updating CycloneDX Core Java to v3.0.5 - currently Dependency-Track in the 593 development branch.

**Comment by stevespringett** [comment]
Closing. Fixed in v4.0 - to be released soon.
